config/env.go

```diff
@@ -54,8 +54,29 @@ func MustReadEnvVar_Boolean(name string) *bool {
 	return ptr.Ptr(value.(bool))
 }
 
+func ReadEnvVarSlice_String(pattern string) []string {
```

Comment:
any reason for breaking with convention and using `_` in method name?

Reply:
Maybe it should also have a suffix of `Must`? Since it doesn't return error and panics, when regex is incorrect? Or... maybe it should accept `re` instead of `string` and then there would be no problem?

Reply:
@Tofel no must since it returns empty slice if no vars found